README.md

```diff
@@ -111,10 +111,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let publisher = service.publisher().create()?;
 
     while !SignalHandler::termination_requested() {
-        let mut sample = publisher.loan()?;
-        unsafe {
-            sample.as_mut_ptr().write(1234);
-        }
+        let sample = publisher.loan_uninit()?;
+        let sample = sample.write_payload(1234);
```

Comment:
Out of curiosity. Let's assume I have a low-level function that requires a slice/pointer to some memory and would write the content into the provided slice. How would I use `loan_uninit` and the sample in such a scenario.

So the function signature would be for instance:
```rust
fn populate_data(data: *mut MyData);
```

Reply:
That would be
```rust
populate_data(sample.payload_mut().as_mut_ptr());
```